lib/config/validation.ts

```diff
@@ -29,7 +29,7 @@ import * as managerValidator from './validation-helpers/managers';
 const options = getOptions();
 
 let optionTypes: Record<string, RenovateOptions['type']>;
-let optionParents: Record<string, RenovateOptions['parent']>;
+let optionParents: Record<string, RenovateOptions['parents']>;
```

Comment:
```suggestion
let optionParents: Record<string, string[]>;
```
it's now a string array

Reply:
☝️ Updated